app/components/pdf_component.html.erb

```diff
@@ -8,13 +8,15 @@
   <% end %>
 
   <% component.with_authorization_messages do %>
-    <%= render LoginComponent.new %>
-    <%= render LocationRestrictionComponent.new %>
-    <%= render CompanionWindows::ContentNotAvailableComponent.new unless viewer.available? %>
+    <div data-controller="iiif-auth-restriction" data-action="auth-denied@window->iiif-auth-restriction#displayMessage needs-login@window->iiif-auth-restriction#displayLoginPrompt auth-success@window->iiif-auth-restriction#hideLoginPrompt thumbnail-clicked@window->iiif-auth-restriction#clearLocationRestriction">
```

Comment:
Perhaps we could have thumbnail-clicked clear all the message panels (Except the one that says "logged in" for a logged in user).  So not just clear location restriction but clearAuthMessages where we hide both login prompt and the location restriction panel

Reply:
Please see the comment above for context.